xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp

```diff
@@ -182,7 +178,21 @@ bool CInputStreamAddon::Open()
   }
   return ret;
 }
-
+void CInputStreamAddon::DetectDisplayResolution()
+{
+  unsigned int videoWidth = 1280;
+  unsigned int videoHeight = 720;
+  if (m_player)
+    m_player->GetVideoResolution(videoWidth, videoHeight);
+  if(videoWidth_!=videoWidth || videoHeight_!=videoHeight)
```

Comment:
Space after the `if` and before the bracket. Space also required around the `!=`'s.

Reply:
@phunkyfish thanks, noted git-clang-format.